dataline-config/persistence/src/test/java/io/dataline/config/persistence/DefaultConfigPersistenceTest.java

```diff
@@ -25,171 +25,105 @@
 package io.dataline.config.persistence;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.fail;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.*;
 
-import com.fasterxml.jackson.databind.JsonNode;
-import com.google.common.collect.ImmutableMap;
-import io.dataline.commons.json.Jsons;
-import io.dataline.config.Column;
-import io.dataline.config.DataType;
+import com.google.common.collect.Sets;
 import io.dataline.config.Schema;
 import io.dataline.config.StandardSource;
 import io.dataline.config.StandardSync;
-import io.dataline.config.Table;
 import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Path;
-import java.util.Collections;
-import java.util.Set;
 import java.util.UUID;
-import org.apache.commons.io.FileUtils;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
 class DefaultConfigPersistenceTest {
 
-  private Path rootPath;
-  private DefaultConfigPersistence configPersistence;
+  public static final UUID UUID_1 = new UUID(0, 1);
```

Comment:
why do you prefer the stati uuids to the randomly generated ones?

Reply:
because they are recognizable when there are failing tests.